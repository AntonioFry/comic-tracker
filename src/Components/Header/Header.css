header {
  background: black;
  display: flex;
  justify-content: center;
  margin: 0px 0px;
}

.marvel-logo {
  width: 175px;
}