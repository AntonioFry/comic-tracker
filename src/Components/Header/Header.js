import React from 'react';
import './Header.css';

export const Header = () => {
  return (
    <header>
      <img className="marvel-logo" src={require('../../../public/marvel-logo.png')} alt="Marvel comics logo" />
    </header>
  )
}