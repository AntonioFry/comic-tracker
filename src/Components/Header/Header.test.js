import React from 'react';
import { render, cleanup } from '@testing-library/react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

import '@testing-library/jest-dom';
import { Header } from './Header.js';
import { BrowserRouter } from 'react-router-dom';

describe('Header', () => {

  it('Should match the snapshot', () => {
    const tree = renderer.create(
      <BrowserRouter>
        <Header />
      </BrowserRouter>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  })
});