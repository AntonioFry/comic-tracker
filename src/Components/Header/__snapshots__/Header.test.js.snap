exports[`Header Should match the snapshot 1`] = `
<header>
  <a
    className="logo-link"
    href="/"
    onClick={[Function]}>
    <img
      alt="Marvel comics logo"
      className="marvel-logo"
      src="marvel-logo.png" />
  </a>
</header>
`;
